### PR TITLE
Updated .css files to improve visual component

### PR DESCRIPTION
**Pull request by AshleyL119**
Updated background colors, button settings and text styles

### CODE REVIEW

File: santoriniWebsite/client/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-scripts": "5.0.1",
+    "react-scripts": "^3.0.1",
```
Comment (RichardJSun) [inline santoriniWebsite/client/package.json:9]: Is there a reason to downgrade the template scripts?

File: santoriniWebsite/client/package.json [hunk 2]
```diff
@@ -42,5 +42,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@babel/plugin-transform-private-property-in-object": "^7.25.7"
```
Comment (RichardJSun) [inline santoriniWebsite/client/package.json:47]: this doesn't seem used and it's also a duplicate key

Comment (RichardJSun) [reply santoriniWebsite/client/package.json:47]: actually it seems to be subdependency?

Comment (AshleyL119) [reply santoriniWebsite/client/package.json:47]: Oh, I haven't committed my latest changes yet and I upgraded back to version 5 and deleted the Babel subdependency . These two changes was bc to troubleshoot errors at the time. 
I did figure out one key change to make the website work tho,
 "scripts": {
    "start": "set NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
    ...
    }
    
    Basically, this flag solved the issue with the start command and made it compile.

**Pull request merged**